tensorflow/core/kernels/slice_op.cc

```diff
@@ -205,6 +205,207 @@ class SliceOp : public OpKernel {
   }
 };
 
+#ifdef INTEL_MKL
+template <typename Device, typename T>
+class MklSliceOp : public OpKernel {
+ public:
+  explicit MklSliceOp(OpKernelConstruction* context) : OpKernel(context) {}
+
+  void Compute(OpKernelContext* context) override {
+    TensorShape output_shape;
+    bool is_identity = true;
```

Comment:
Looks like a lot of shared code with SliceOp above. Can this op inherit from SliceOp?